client/question/question.js

```diff
@@ -248,7 +248,7 @@ tie.constant('WRONG_LANGUAGE_ERRORS', {
   }, {
     // Used the Java comment syntax
     errorName: 'javaComment',
-    regExString: '\\/(\\s*|\\w*)*\\n|\\/\\*(\\*)?(\\s*|\\w*)*\\*\\/',
+    regExString: '\\/\\/(\\s*|\\w*)*\\n|\\/\\*(\\*)?(\\s*|\\w*)*\\*\\/',
```

Comment:
Btw, are the two inner asterisks in `(\\s*|\\w*)*` (both occurrences) necessary? (Wouldn't the asterisk outside the parens take care of that?)